telegram/ext/filters.py

```diff
@@ -183,7 +184,15 @@ def __init__(self, name: str = None, data_filter: bool = False):
         self._data_filter = data_filter
 
     def check_update(self, update: Update) -> Optional[Union[bool, DataDict]]:  # skipcq: PYL-R0201
-        """Checks if the specified update is a message."""
+        """
+        Checks if the specified update is a message.
+
+        Args:
+            update (:class:`telegram.Update`): Incoming update.
+
+        Returns:
+            :obj:`bool` | :obj:`Dict[str, list]`
```

Comment:
```suggestion
            :obj:`bool` | Dict[str, list]
```
sorry, forgot that the previous one fails the docs